frontend/src/app/_components/responsiveLinks/ResponsiveLinks.tsx

```diff
@@ -0,0 +1,49 @@
+import Link from 'next/link'
+import { usePathname } from 'next/navigation'
+import styles from './ResponsiveLinks.module.css'
+
+interface ResponsiveLinksProps {
+  isNavVisible: boolean
+  toggleNav: () => void
+}
+
+const ResponsiveLinks = ({ isNavVisible, toggleNav }: ResponsiveLinksProps) => {
```

Comment:
The logo and the company name (postbox library) are also navigation links, so I thought responsive links fit better, what about drop down links or hamburger links?

Reply:
I think hamburger nav links would be good. With drop down links it's a bit like responsive links, it feels a bit generic and could get confused with something else on the site - if you read that component name on it's own you don't know that it's not actually reusable for any group of responsive links. I also think it might be worth renaming your state as well. The nav is always visible, it's the type of nav that is visible or not. What about isHamburgerNav visibile?

Reply:
cool I'm going to change this to hamburger nav links and I agree about the state that makes sense :) I'll update that too